README.md

```diff
@@ -1,27 +1,43 @@
-# Feature Extraction and Reverse Image Search on Caltech-101 Dataset
-This notebook will guide you through the process of analyzing an image dataset using a pre-trained convolution network (VGG16) and extracting feature vectors for each image
+# Accelerate Reverse Image Search with GPU for feature extraction
 
-Post analysis we try to demonstrate 'reverse image search' one of the widely popular applications of image analysis.
+In this code pattern, we will guide you through the process of analyzing an image dataset using a pre-trained convolution network (VGG16) and extracting feature vectors for each image using a Jupyter notebook.
+This is a computationally expensive process, requiring over 300 times longer on a CPU versus a GPU. We'll use the GPU environment on Watson Studio or on your local machine to accelerate feature extraction.
```

Comment:
this first sentence is confusing - do you mean "takes 300 time longer"?

Reply:
line 4

Reply:
fixed